README.md

```diff
@@ -35,9 +36,11 @@ cd clickbench/
 # Download the dataset
 bash setup.sh
 
-# The results are written to tpch_datafusion.csv and tpch_duckdb.csv
-# Run and plot comparison benchmarks
-bash benchmark.sh single single
+# The results are written to
```

Comment:
I am envisioning that each time we run the benchmarks and it gathers data, the data will be written into the `results` directory